Example/Info.plist

```diff
@@ -45,5 +45,22 @@
 	</array>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Demo app helps you find more information near you</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>localhost</key>
```

Comment:
Is there an alternative to using a self-signed cert?
Can we get a real cert from a CA?

Reply:
yes we can, but would it be unnecessary for our needs ?

Reply:
@donnie-jp , if we can get a real cert from CA that would be awesome. I created it from my Keychain assistant. How do we get one?

Reply:
But the private key needs to be committed to this repo, right? So the CA would revoke our certificate because the private key has been exposed.

Reply:
Sorry, I don't know the details - just flagging it as a possible issue.

Reply:
Can we inject the certificate directly into the build process and not include it in the repo?

Reply:
Injecting it at build time might be better.

Or even better, is it possible to generate the certificate at runtime on the device? This is possible on Android, but not sure about iOS...